README.md

```diff
@@ -12,16 +12,18 @@
 This plugin provides simple methods store and
 sync data. It combines [pouchdb-hoodie-api](https://github.com/hoodiehq/pouchdb-hoodie-api) and [pouchdb-hoodie-sync](https://github.com/hoodiehq/pouchdb-hoodie-sync).
 
+## Example
+
+    var Store = require('hoodie-client-store')
+    var store = new Store('mydbname', { remote: 'http://localhost:5984/mydbname' })
+    // or
+    Store.defaults({remoteBaseUrl: 'http://localhost:5984' })
+    var store = new Store('mydb')
+
```

Comment:
can you please use `\`\`\`js` instead of the 4 space indentation, for the JavaScript syntax highlighting?